tests/unit/qunit/filter-test-modules-test.js

```diff
@@ -0,0 +1,153 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import { convertFilePathToModulePath, filterTestModules } from 'ember-exam/test-support/-private/filter-test-modules';
+
+module('Unit | filter-test-modules', function(hooks) {
+  setupTest(hooks);
+
+  module('covertFilePathToModulePath', function(hooks) {
+    setupTest(hooks);
+
+    test('should return an input string without file extension when the input contains file extension', function(assert) {
+      assert.equal('/tests/integration/foo', convertFilePathToModulePath('/tests/integration/foo.js'))
```

Comment:
equal's params are `actual, expected`, so i think this should be:
`assert.equal( convertFilePathToModulePath('/tests/integration/foo.js'), '/tests/integration/foo')`
https://api.qunitjs.com/assert/equal

also, missing semicolon at the end